lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runtime/azbatch/util/AzureStorageUtil.java

```diff
@@ -1,153 +0,0 @@
-/*
```

Comment:
Suggest using "git mv" to keep the history in the future.

Reply:
good call. I was actually doing the refactoring using IntelliJ. I'll see if I can fix this.

Reply:
No idea why this is happening. I did "git reset HEAD~1 --soft", moved the files around and the "git mv". Still, it's showing up as delete + add. Strangely, all other files that I refactored are showed as having been moved.